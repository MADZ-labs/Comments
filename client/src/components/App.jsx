import React, { Component } from 'react';
import styled from 'styled-components';
import Comments from './Comments';
import '../stylesheets/main.scss';

const Container = styled.div`
  display: flex;
  justify-content: center;
  max-width: 1040px;
  padding-right: 1.8rem !important;
  padding-left: 1.8rem !important;
  margin-right: auto;
  margin-left: auto;
`;

export default class App extends Component {
  constructor(props) {
    super(props);
    this.state = {

    };
  }

  render() {
    return (
      <section>
        <h1>Comment App</h1>
        <Container className='container'>
          <Comments />
        </Container>
      </section>
    );
  }
}

