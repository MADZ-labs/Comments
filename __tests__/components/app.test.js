import React from "react";
import { shallow, mount, render } from "enzyme";
import App from '../../client/src/components/App'

// describe("Index Component", () => {
//   it("should render without throwing an error", () => {
    
//   });
// });
