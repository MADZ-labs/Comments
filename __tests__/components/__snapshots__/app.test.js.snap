// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App Component should render without throwing an error 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <App />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <h1>
          Comment App
        </h1>,
        <styled.div
          className="container"
        >
          <Comments />
        </styled.div>,
      ],
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Comment App",
        },
        "ref": null,
        "rendered": "Comment App",
        "type": "h1",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "children": <Comments />,
          "className": "container",
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {},
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        "type": [Function],
      },
    ],
    "type": "section",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <h1>
            Comment App
          </h1>,
          <styled.div
            className="container"
          >
            <Comments />
          </styled.div>,
        ],
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Comment App",
          },
          "ref": null,
          "rendered": "Comment App",
          "type": "h1",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "children": <Comments />,
            "className": "container",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {},
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
          "type": [Function],
        },
      ],
      "type": "section",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
